.github/workflows/ci-pipeline.yml

```diff
@@ -27,8 +27,8 @@ env:
   CONTAINER_REGISTRY: ghcr.io
   LOCAL_REGISTRY: registry.localhost:5000
 
-  CARAML_CHARTS_REPOSITORY: romanwozniak/helm-charts
-  CARAML_CHARTS_REF: feat-mlp-apps-configuration
+  CARAML_CHARTS_REPOSITORY: caraml-dev/helm-charts
+  CARAML_CHARTS_REF: main
```

Comment:
Made this change because the PR mentioned in this thread [here](https://github.com/gojek/mlp/pull/71#discussion_r1028267478) has been merged.